city_scrapers/spiders/il_aging_advisory_council.py

```diff
@@ -0,0 +1,113 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import ADVISORY_COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlAgingAdvisoryCouncilSpider(CityScrapersSpider):
+    name = "il_aging_advisory_council"
+    agency = "Illinois Aging Advisory Council"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www2.illinois.gov/aging/PartnersProviders/OlderAdult/Pages/acmeetings.aspx"
+    ]
+    location = {
+        'address': {
+            'Springfield': 'One Natural Resources Way #100',
+            'Chicago': '160 N. LaSalle Street, 7th Floor'
+        },
+        'name': {
+            'Springfield': 'Illinois Department on Aging',
+            'Chicago': 'Michael A. Bilandic Building'
+        }
+    },
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        self._validate_location(response)
+
+        self._validate_meeting_times(response)
+
+        valid_date_list = self._create_valid_date_list(response)
+
+        if valid_date_list:
+            for item in valid_date_list:
+                meeting = Meeting(
+                    title='Illinois Department on Aging Advisory Committee Meetings',
+                    description='Meetings are open to the public',
```

Comment:
Generally the `description` is only filled in if there's something dynamic on the page, so this can be an empty string

Reply:
Updated description to empty string